api/api.go

```diff
@@ -61,6 +61,7 @@ const (
 	SpecExportOptions        = "export_options"
 	SpecExportOptionsEmpty   = "empty_export_options"
 	SpecMountOptions         = "mount_options"
+	SpecCSIMountOptions      = "csi_mount_options"
```

Comment:
Out of curiosity, why didn't we do some sort of merge between CSI and <wherever else mount options can come from> and store them in the same field? I can't think of any reason off the top of my head for why we'd want to treat them differently, but I could definitely have just missed a conversation.

Reply:
@ggriffiths has a comment about this, ping you there.